step-capstone/src/components/Trip.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import TravelObject from './TravelObject'
-import { Box, Grid, Typography } from '@material-ui/core'
+import { Grid } from '@material-ui/core'
 
 export default class Trip extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
             data: []
         }
+
+        this.deleteObject = this.deleteObject.bind(this);
```

Comment:
Just a question out of curiosity, what does this line of code mean, Emmie?

Reply:
I guess its purpose is to pass down the function the sub component: Travel Object? I still wonder what "bind" does.

Reply:
Basically the "this" keyword doesn't work as it does it java as we're used to, and binding "this" to the function ensures that anytime this is called (for example, passing down this.function), it's referring to this class component. A common work around is using arrow functions, but I think "this binding" is generally preferred.

Reply:
Learn more here: https://reactjs.org/docs/faq-functions.html